haystack/nodes/search_engine/web.py

```diff
@@ -63,17 +63,24 @@ def run(
         labels: Optional[MultiLabel] = None,
         documents: Optional[List[Document]] = None,
         meta: Optional[dict] = None,
+        top_k: Optional[int] = None,
     ) -> Tuple[Dict, str]:
         """
         Search the search engine for the given query and return the results. Only the query parameter is used.
```

Comment:
Let's change "Only the query parameter is used." to "Only the query parameter and the top_k parameter are used."